src/content/docs/usage/targets.mdx

```diff
@@ -0,0 +1,313 @@
+---
+title: Targets
+description: Learn how to manage Target configuration across Workspaces.
+sidebar:
+  label: Targets
+---
+
+import Label from '@components/Label.astro'
+
+<Label>
+  Distribution: **Open Source**
+</Label>
+
+A Target is an isolated processing unit where Workspaces are executed. It can operate on various infrastructures, including cloud-based and on-premises environments. Targets provide the necessary compute resources and meet security requirements within the defined infrastructure.
+
+## Create a Target
+
+Daytona provides an option to create a Target. Creating a Target requires creating a Target Configuration and specifying the Provider to use.
+
+1. Run the following command to create a Target:
+
+```shell
+daytona target create
+```
+
+:::note
+If you have not previously installed a Provider, registered a Runner, or created a Target Configuration, Dayona will prompt you to complete these steps before creating a Target.
+:::
+
+<br />
+
+2. Select the appropriate Provider for the environment you want to deploy to.
+
+```text
+Choose a provider
+4 items
+===
+Docker
+v0.0.0
+DigitalOcean
+v0.0.0
+AWS
+v0.0.0
+Azure
+v0.0.0
+GCP
+v0.0.0
+Hetzner
+v0.0.0
+Fly
+v0.0.0
+```
+
+:::note
+If you have not previously installed a [Runner](/docs/usage/runners), Daytona will prompt you select a Runner to manage the Provider. Upon selecting, Daytona will install the Runner on the target system and connect it to the Daytona Server.
+:::
+
+<br />
+
+3. Enter a name for your Target Configuration.
+
+```text
+Target Config Name
+>
+```
+
+4. Enter the appropriate configuration options when prompted. The configuration options vary based on the selected Provider. The following example shows adding a local Docker Target Configuration.
```

Comment:
Remote docker TC.

But let's actually go for the local TC and have `Sock Path` to save on vertical space